platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -354,7 +354,7 @@ In style specification | Method, function, or predicate type | Format string syn
 `>=`                   | `NSGreaterThanOrEqualToPredicateOperatorType` | `key >= value`
 `all`                  | `NSAndPredicateType` | `p0 AND … AND pn`
 `any`                  | `NSOrPredicateType` | `p0 OR … OR pn`
-`case`                 | `MGL_IF` | `MGL_IF(1 = 2, YES, 2 = 2, YES, NO)` 
+`case`                 | `+[NSExpression expressionForConditional:trueExpression:falseExpression:]` or `MGL_IF` | `TERNARY(1 = 2, YES, NO)` or `MGL_IF(1 = 2, YES, 2 = 2, YES, NO)`
```

Comment:
I restored full support for `+[NSExpression expressionForConditional:trueExpression:falseExpression:]` where available, along with the tests for it. On iOS 9.0 and above, we convert `case` to `TERNARY()` when it has only one conditional. This is the preferred syntax, because you can inline the predicate instead of wrapping it in a constant value expression, which means you can write a conditional in a single format string.